src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -517,6 +517,12 @@ private void AddTupleTypeName(INamedTypeSymbol symbol)
         {
             Debug.Assert(symbol.IsTupleType);
 
+            if (this.format.MiscellaneousOptions.IncludesOption(SymbolDisplayMiscellaneousOptions.DoNotExpandTupleType))
+            {
+                builder.Add(CreatePart(SymbolDisplayPartKind.StructName, symbol, "<tuple>"));
```

Comment:
this is similar to what we do with anonymous types.  With anon-types, the compiler just makes a single part, allowing the IDE control of how to build the final display string.  For tuples it's fine (by default) to expand it as `(...)`, but we need a way to have that not happen in order to use the new display logic.